napari/utils/tree/group.py

```diff
@@ -70,17 +78,17 @@ def __newlike__(self, iterable: Iterable):
         new._list.extend(iterable)
         return new
 
-    def __getitem__(self, key) -> Union[NodeType, Group[NodeType]]:
+    def __getitem__(self, key) -> Union[NodeType, Group[NodeType]]:  # type: ignore [override]
         return super().__getitem__(key)
```

Comment:
```suggestion
```
Given this just calls `super().__getitem__`, I think it's safe to get rid of it and rely on typing from the parent class?

Reply:
but does the super class type annotation resolve to proper type?

Reply:
No, it missed `Group[NodeType]`

Technically mypy does not complain so we could remove it until we actually get an error.